compiler/damlc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -171,19 +107,4 @@ instance IsParties [Party] where
 
 instance IsParties (Optional Party) where
   toParties None = []
-  toParties (Some p) = [p]
-
-class Template c => TemplateKey c k | c -> k where
-    -- | The key of a contract.
-    key : c -> k
-
-    -- | The maintainers of the contract key.
-    maintainer : k -> [Party]
-
-    -- | HIDE
-    internalFetchByKey : k -> Update (Pair "contractId" "contract" (ContractId c) c)
-    internalFetchByKey = magic @"fetchByKey"
-
-    -- | HIDE
-    internalLookupByKey : k -> Update (Optional (ContractId c))
-    internalLookupByKey = magic @"lookupByKey"
+  toParties (Some p) = [p]
```

Comment:
Can you please put
```
"files.trimFinalNewlines": true,
"files.insertFinalNewline": true,
```
into your VS Code config to avoid removing the trailing newlines.